AsyncDisplayKit/ASTableView.mm

```diff
@@ -1553,6 +1553,11 @@ - (void)nodeHighlightedStateDidChange:(ASCellNode *)node
   }
 }
 
+- (ASSizeRange)constrainedSizeForNode:(ASCellNode *)node
+{
+  return [self dataController:self.dataController constrainedSizeForNodeAtIndexPath:[self indexPathForNode:node]];
```

Comment:
@maicki it's still in the diff's current version, are there other updates to push?